core/src/main/java/io/confluent/rest/SslFactory.java

```diff
@@ -16,29 +16,97 @@
 
 package io.confluent.rest;
 
-import java.nio.file.Path;
```

Comment:
@smjn What is the role out plan for the pem keystores in prod to make sure REST doesn't break?

Reply:
Also is this change backwards compatible? Say in "release 10" this change is released to prod, but for reason(bug) the REST is rolled back to "release 9", would it be safe?

Reply:
@msn-tldr 
Pem support is additional (on top of current pkcs and jks). Current functionality should remain exactly same. Only when specific NEW Fips clusters are provisioned should the pem codeflow get invoked. This depends on the value of `ssl.provider`. Using Sunjsse or non-fips providers, the codeflow should remain same.